cmd/oras/internal/option/remote.go

```diff
@@ -118,17 +122,29 @@ func (opts *Remote) ApplyFlagsWithPrefix(fs *pflag.FlagSet, prefix, description
 
 // Parse tries to read password with optional cmd prompt.
 func (opts *Remote) Parse() error {
+	if opts.IdentityToken != "" || opts.IdentityTokenFromStdin {
```

Comment:
The returned error should be specific about which two flags are conflicting with each other.

Reply:
implemented specific error messages